<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Isolines</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <column>
           <property name="text">
            <string>Объекты:</string>
           </property>
           <property name="font">
            <font>
             <pointsize>6</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Точка</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/icons/icons8-map-pin-50.png</normaloff>:/icons/icons/icons8-map-pin-50.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Группа</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/icons/icons8-group-objects-50.png</normaloff>:/icons/icons/icons8-group-objects-50.png</iconset>
           </property>
           <item>
            <property name="text">
             <string>Надпись</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/icons8-text-50.png</normaloff>:/icons/icons/icons8-text-50.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Линия</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/icons8-line-50.png</normaloff>:/icons/icons/icons8-line-50.png</iconset>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Экспортировать как...</string>
     </property>
     <addaction name="actionJPEG"/>
     <addaction name="actionPNG"/>
     <addaction name="actionGIF"/>
     <addaction name="actionRAW"/>
    </widget>
    <addaction name="Open"/>
    <addaction name="separator"/>
    <addaction name="Save"/>
    <addaction name="SaveAs"/>
    <addaction name="separator"/>
    <addaction name="ExportDFX"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="Close"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="dot"/>
    <addaction name="line"/>
    <addaction name="label"/>
    <addaction name="separator"/>
    <addaction name="group"/>
    <addaction name="ungroup"/>
    <addaction name="separator"/>
    <addaction name="delete_2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Модель</string>
    </property>
    <addaction name="table"/>
    <addaction name="separator"/>
    <addaction name="generate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Open"/>
   <addaction name="separator"/>
   <addaction name="Save"/>
   <addaction name="SaveAs"/>
   <addaction name="separator"/>
   <addaction name="ExportDFX"/>
   <addaction name="actionJPEG"/>
   <addaction name="actionPNG"/>
   <addaction name="actionGIF"/>
   <addaction name="actionRAW"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="table"/>
   <addaction name="generate"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="dot"/>
   <addaction name="line"/>
   <addaction name="label"/>
   <addaction name="separator"/>
   <addaction name="group"/>
   <addaction name="ungroup"/>
   <addaction name="separator"/>
   <addaction name="delete_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="Open">
   <property name="text">
    <string>Открыть..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Save">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="ExportDFX">
   <property name="text">
    <string>Экспортировать в dxf</string>
   </property>
  </action>
  <action name="actionJPEG">
   <property name="text">
    <string>JPEG</string>
   </property>
   <property name="toolTip">
    <string>Экспортировать в JPEG</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
   <property name="toolTip">
    <string>Экспортировать в PNG</string>
   </property>
  </action>
  <action name="actionGIF">
   <property name="text">
    <string>GIF</string>
   </property>
   <property name="toolTip">
    <string>Экспортировать в GIF</string>
   </property>
  </action>
  <action name="Close">
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="dot">
   <property name="text">
    <string>Точка</string>
   </property>
  </action>
  <action name="label">
   <property name="text">
    <string>Надпись</string>
   </property>
  </action>
  <action name="table">
   <property name="text">
    <string>Таблица</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionRAW">
   <property name="text">
    <string>RAW</string>
   </property>
   <property name="toolTip">
    <string>Экспортировать в RAW</string>
   </property>
  </action>
  <action name="line">
   <property name="text">
    <string>Линия</string>
   </property>
  </action>
  <action name="generate">
   <property name="text">
    <string>Построить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="group">
   <property name="text">
    <string>Сгруппировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="ungroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Разгруппировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="delete_2">
   <property name="text">
    <string>Удалить объект</string>
   </property>
  </action>
  <action name="color">
   <property name="text">
    <string>Цвет</string>
   </property>
   <property name="toolTip">
    <string>Выбор цвета</string>
   </property>
  </action>
  <action name="hide">
   <property name="text">
    <string>Скрыть/Раскрыть</string>
   </property>
   <property name="toolTip">
    <string>Скрыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="fix">
   <property name="text">
    <string>Зафиксировать/Отфиксировать</string>
   </property>
   <property name="toolTip">
    <string>Закрепить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
